python/federatedml/secureprotol/secret_sharing/vss.py

```diff
@@ -0,0 +1,69 @@
+import random
+from federatedml.secureprotol import gmpy_math
+
+
+class Vss(object):
```

Comment:
We follow CamelCase convention for class names. Suggested name: VerifiableSecretSharing.